src/components/atoms/Radio/styles.css

```diff
@@ -0,0 +1,53 @@
+.radio {
+  display: inline-block;
+  position: relative;
+}
+
+.input {
+  position: relative;
+  width: 22px;
+  height: 22px;
+
+  &::before,
+  &::after {
+    content: '';
+    position: absolute;
+    border-radius: 100%;
+    transition: all .3s ease-in-out;
+  }
+
+  &::before {
+    width: 18px;
+    height: 18px;
+    background: var(--color-blue);
+    border: 1px solid var(--color-blue);
+    top: 1px;
+    left: 1px;
+    outline: 0;
+  }
+
+  &::after {
+    width: 5px;
+    height: 5px;
+    background: white;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    opacity: 0;
+  }
+
+  &:focus {
+    border-color: var(--color-focusable-blue);
```

Comment:
I'm don't agree with this approach of using colors named variable, it should be named based on its role and functionality, like just: `--input-focused`, because in the future if anyone wants to change the focus color to red they'll have to change the variable name and edit all components.